broker-daemon/interchain/index.js

```diff
@@ -0,0 +1,275 @@
+const { logger } = require('../utils')
```

Comment:
Is it still beneficial to have `interchain` be it's own top-level module? Is it only going to be used by the order state machine? If so, we maybe could make this a file in the state-machines module. As a side note, I think conventionally `index.js` usually just presents exports for the public interface of the module.